dis_snek/api/http/http_client.py

```diff
@@ -137,9 +137,9 @@ class HTTPClient(
 ):
     """A http client for sending requests to the Discord API."""
 
-    def __init__(self, connector: Optional[BaseConnector] = None, loop: Optional[asyncio.AbstractEventLoop] = None):
+    # def __init__(self, connector: Optional[BaseConnector] = None, loop: Optional[asyncio.AbstractEventLoop] = None):
```

Comment:
You should probably remove this comment, we don't want it in the final repository when this pull request is merged.